package.json

```diff
@@ -191,5 +191,8 @@
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
+  "resolutions": {
```

Comment:
This is required since `redux-notifications` has `react-redux v4` as a dependency.